eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml

```diff
@@ -0,0 +1,17 @@
+eight_points_guzzle:
+    clients:
+        my_client:
+            # Write here the host where to do requests
+            base_url: "http://target.url"
```

Comment:
You like this line and I agree it is important. But lets make it a comment and this service will be useable without the user has to change the config.

Reply:
A leave `base_url` here but commented it. Is it ok?